libs/dh/shared/data-access-mocks/src/lib/admin.ts

```diff
@@ -189,10 +190,20 @@ function getUserOverviewQuery() {
   });
 }
 
-function putMarketParticipantUserRoleUpdate(apiBase: string) {
-  return http.put(`${apiBase}/v1/MarketParticipantUserRole/Update`, async () => {
+function updateUserRoleMutation() {
+  return mockUpdateUserRoleMutation(async () => {
     await delay(mswConfig.delay);
-    return new HttpResponse(null, { status: 200 });
+
+    return HttpResponse.json({
```

Comment:
Suggestion: I have started to look at the variables and ex. if id = 2 then make it fail. Then it is easier to the error handling when using mock data

Reply:
Tried to include an error state as well.